src/v2/models/XcmModels.ts

```diff
@@ -14,6 +14,7 @@ export enum Chain {
   STATEMINT = 'Statemint',
   KINTSUGI = 'Kintsugi',
   INTERLAY = 'Interlay',
+  CRUST_SHADOW = 'Crust-shadow',
```

Comment:
Can we do Crust Shadow?

Reply:
I've resolved it (for UI) in this PR.
https://github.com/AstarNetwork/astar-apps/pull/607

> fix: cast the network name from 'Crust-shadow' to 'Crust Shadow' (for UI)

Reply:
CC: @fiexer